### ISSUE TITLE
An suggestion to configure cooper process in project.clj

### ISSUE THREAD

**Issue by DogLooksGood** [kouphax/lein-cooper#3]
How about this?

```clojure
:cooper {"cljsbuild" ["lein" "cljsbuild" "auto"]
         "garden"    ["lein" "garden" "auto"]
         "repl"      ["lein" "repl" ":headless"]}
```
Then start with
```shell
$ lein cooper garden repl
```
to lanuch both `garden` and `repl` task, `cljsbuild` is skipped.

Short the command with :aliases
```clojure
:aliases {"dev" ["cooper" "garden" "repl"]}
```

I've made a little change for this on my fork, may I send a PR?

**Comment by kouphax** [comment]
Yes please send a PR.

I have a few questions.

1. Does this handle non leiningen/clojure processes?  I assume it does but wanted to check.
2. Are you suggesting we get rid of the Procfile?  If so do you see a problem with that approach?

Actually only 2 questions really.

**Comment by DogLooksGood** [comment]
Here's the change:
ee7ce0119e34bc27dd77d10f773691a1cddee5e9

1. I only add additional logic to read configuration from `:cooper` in project.clj if that exists. nothing change on the way of handling processes.
2. If we get rid of `Procfile`, we can't use it out of project. But it's really necessary to have multiple profiles . Instead of specifying the procfile's name, putting configuration in project.clj is more  idiomatic.

**Comment by kouphax** [comment]
OK let me have a mull over #4.  Thanks for this - it looks good.

**Comment by kouphax** [comment]
Fixed by #4
